server/controllers/deletePostController.js

```diff
@@ -0,0 +1,11 @@
+const { deletePostQuery } = require("../database/queires")
+
+
+const deletePost = (req, res, next) => {
+    const { id } = req.body
+    deletePostQuery(id)
+        .then(data => res.json(data))
+        .catch(err => next(err))
+};
+
+module.exports = deletePost;
```

Comment:
Here, you let the users delete any post they want, even if it's not their post. 
So you can add a post and then I delete it.